src/group/controller/collection.ts

```diff
@@ -23,4 +23,4 @@ class GroupMemberCollectionControlller extends Controller {
 
```

Comment:
The most important endpoint is probably the `put` on this resource, letting us update the list of members. This is more consistent with everything else. The form is also nice though, as it lets us easily make changes.

Reply:
This has not been addressed yet. We usually update resources with `PUT` and the request body should match what we got with `GET`